### ISSUE TITLE
Other ajax api

### ISSUE THREAD

**Issue by ruor** [mtth/azkaban#36]
checked from https://azkaban.readthedocs.io/en/latest/ajaxApi.html#, I found that there were several api that were not included in this python cli, such as fetchProjectLogs, fetchprojectflows, fetchflowgraph. I thinks they're also important. So why we don't have them and is there any plan to add them?

**Comment by mtth** [comment]
These APIs didn't exist at the time this project was created. No plans to add them at this time, but PRs are welcome!

**Comment by ruor** [comment]
thanks! I'm trying to pull request, but it returns 403 when i push to master.
could you please help to check this?
remote: Permission to mtth/azkaban.git denied to ruor.
fatal: unable to access 'https://github.com/mtth/azkaban.git/': The requested URL returned error: 403

**Comment by mtth** [comment]
Yes, you should fork the repo.[This article](https://help.github.com/en/articles/creating-a-pull-request) has a nice overview of the process.

**Comment by ruor** [comment]
thanks! I pulled a request, could you please help to review it when you have time?

https://github.com/mtth/azkaban/pull/37
